apps/web/src/app/[locale]/_components/countdown-timer.tsx

```diff
@@ -43,22 +43,45 @@ export const CountdownTimer = () => {
     return () => {
       clearInterval(timerId);
     };
-  }, [remainingTime, releaseDateTimeInMilliSeconds]);
+  }, [remainingTime, releaseDate]);
 
   const { days, hours, minutes, seconds } = calculateTimeComponents(remainingTime);
 
   if (remainingTime === 0) return null;
 
   return (
-    <div className="flex gap-4 font-bold tabular-nums">
-      <DateCard date={days} label="Days" />
-      <DateCard date={hours} label="Hours" />
-      <DateCard date={minutes} label="Minutes" />
-      <DateCard date={seconds} label="Seconds" />
-    </div>
+    <>
+      <p className="text-center text-xl font-semibold">
+        The next type challenge will unlock at <span className="text-primary">midnight(est)</span>{' '}
+        December {appendNumberSuffix(Math.min(releaseDate.getDate() + 1, 25))}
```

Comment:
That was one thing that was throwing me off - because without the +1, it was saying December 2nd (observed at 12:30am est on the 2nd), even though it's already December 2nd and should be saying the 3rd, right?

Reply:
Oh, let me take a look. I think that's because it doesn't update until it enters the useEffect. I'll fix it!

Reply:
Yep! Added it, totally forgot to add it because I'm writing all of this from my phone. Sorry about that